modules/ROOT/partials/configuration/textpattern_patterns.adoc

```diff
@@ -107,7 +108,7 @@ tinymce.init({
 
 Using the configuration in this example:
 
-* `{start: '#', format: 'h1'}` - Typing `#`, some text, and then pressing `Enter` will convert the text to a `h1` heading.
+* `{start: '\#', format: 'h1'}` - Typing `#`, some text, and then pressing `Enter` will convert the text to a `h1` heading.
```

Comment:
```suggestion
* `+{start: '#', format: 'h1'}+` - Typing `#`, some text, and then pressing `Enter` will convert the text to a `h1` heading.
```
This is probably better solved by using literal monospace markup instead of escaping certain characters.